parlai/core/agents.py

```diff
@@ -682,7 +682,7 @@ def create_task_agent_from_taskname(opt):
                            '--task {task_name}.')
     if not opt.get('task'):
         opt['task'] = 'pytorch_teacher'
-    if ',' not in opt['task']:
+    if ',' not in opt['task'] and 'MultiTaskTeacher' not in opt['task']:
```

Comment:
Can you describe the setting that this workaround fixes? It feels like a bandaid, rather than a root cause.

Reply:
Yeah, I agree. That this is a bandaid instead of a fix -- What should be the expected behaviour of calling <task_name>:MultiTaskTeacher?

Reply:
Reverting this to address separately.